client/src/domain/home/search.module.scss

```diff
@@ -2,7 +2,7 @@
 @import "layout";
 
 .searchContainer {
-  margin-top: 5rem;
+  margin-top: -5.5rem;
```

Comment:
Would it be possible to make this adjustment in the component that uses `<Search />` instead? Or is this needed for something that this component caused?